app/models/calculated_handover_date.rb

```diff
@@ -8,6 +8,10 @@ class CalculatedHandoverDate < ApplicationRecord
   REASONS = {
     com_responsibility: 'COM Responsibility',
     recall_case: 'Recall case',
+    recall_release_soon: 'Recall case',
```

Comment:
Do these values need to be more descriptive of their keys? 🤔
I guess depends where this is used and if it is user facing text or just internal.